src/panels/lovelace/common/compute-card-size.ts

```diff
@@ -1,5 +1,13 @@
 import { LovelaceCard } from "../types";
 
-export const computeCardSize = (card: LovelaceCard): number => {
-  return typeof card.getCardSize === "function" ? card.getCardSize() : 4;
+export const computeCardSize = async (card: LovelaceCard): Promise<number> => {
+  if (typeof card.getCardSize === "function") {
+    return card.getCardSize();
+  }
+  if (customElements.get(card.localName)) {
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/API/Node/localName:

![image](https://user-images.githubusercontent.com/1444314/82764479-d7a01e80-9dc3-11ea-9217-6b85acef1f99.png)

```suggestion
  if (customElements.get(card.tagName)) {
```

Reply:
That's node, it was moved to element

Reply:
But tagName to lower case would also work